README.md

```diff
@@ -185,6 +185,7 @@ Over 20 models have been optimized/verified on `bigdl-llm`, including *LLaMA/LLa
 | RWKV5 |  | [link](python/llm/example/GPU/HF-Transformers-AutoModels/Model/rwkv5) |
 | Bark | [link](python/llm/example/CPU/PyTorch-Models/Model/bark) | [link](python/llm/example/GPU/PyTorch-Models/Model/bark) |
 | SpeechT5 |  | [link](python/llm/example/GPU/PyTorch-Models/Model/speech-t5) |
+| Qwen1.5-7B-Chat | [link](python/llm/example/CPU/HF-Transformers-AutoModels/Model/qwen1.5-7b) | [link](python/llm/example/GPU/HF-Transformers-AutoModels/Model/qwen1.5-7b) |
```

Comment:
Qwen1.5-7B-Chat -> Qwen1.5, and using `qwen1.5` as the example folder name may be better.

Reply:
Yes, the folder name and the README.md have been updated.